$OpenBSD: patch-scripts_ssvnc,v 1.2 2009/06/23 20:57:46 sthen Exp $

use the correct wish binary first.  N.B. update-patches will destroy
the ${MODTK_BIN} expansion in the 'for' line.

--- scripts/ssvnc.orig	Tue Jan 13 03:02:08 2009
+++ scripts/ssvnc	Tue Jan 13 03:04:45 2009
@@ -30,7 +30,7 @@ fi
 
 if [ "X$WISH" = "X" ]; then
 	WISH=wish
-	for try in wish8.4 wish wish8.3 wish8.5 wish8.6
+	for try in ${MODTK_BIN} wish8.4 wish wish8.3 wish8.5 wish8.6
 	do
 		if type $try > /dev/null; then
 			WISH=$try
